.sauce/config.yml

```diff
@@ -16,7 +16,7 @@ docker:
   # Affects how test files are transferred to the docker container when using the docker mode.
   fileTransfer: copy # Choose between mount|copy.
 playwright:
-  version: 1.10.0 # See https://docs.saucelabs.com/testrunner-toolkit for a list of supported versions.
+  version: 1.11.1 # See https://docs.saucelabs.com/testrunner-toolkit for a list of supported versions.
```

Comment:
Please update the version of the nested examples as well ⚠️

Reply:
updated.